scripts/templates/template.mdx.hbs

```diff
@@ -0,0 +1,39 @@
+---
+title: {{templateName}}
+description: Template for {{templateName}}
+keywords: ['{{templateName}}', 'template']
+hide_table_of_contents: true
+draft: true
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import { Page } from '@site/src/components/templates/Page/Page';
+import { CodeCollapse } from '@site/src/components/templates/CodeCollapse';
```

Comment:
This two components are added in https://github.com/grafana/design-system/pull/89.

Reply:
Have you already merged main? I tested it locally and it triggered some errors that may be related to the branch not being updated 🤔